specification/peering/resource-manager/Microsoft.Peering/stable/2022-01-01/peering.json

```
@@ -0,0 +1,4141 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all the cdn peering prefixes advertised at a particular peering location": {
+            "$ref": "./examples/ListCdnPeeringPrefixes.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/checkServiceProviderAvailability": {
+      "post": {
+        "tags": [
+          "CheckServiceProviderAvailability"
+        ],
+        "description": "Checks if the peering service provider is present within 1000 miles of customer's location",
+        "operationId": "CheckServiceProviderAvailability",
+        "parameters": [
+          {
+            "name": "checkServiceProviderAvailabilityInput",
+            "in": "body",
+            "description": "The CheckServiceProviderAvailabilityInput indicating customer location and service provider.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckServiceProviderAvailabilityInput"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "enum": [
+                "Available",
+                "Unavailable"
+              ],
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check if peering service provider is available in customer location": {
+            "$ref": "./examples/CheckServiceProviderAvailability.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/legacyPeerings": {
+      "get": {
+        "tags": [
+          "LegacyPeerings"
+        ],
+        "description": "Lists all of the legacy peerings under the given subscription matching the specified kind and location.",
+        "operationId": "LegacyPeerings_List",
+        "parameters": [
+          {
+            "name": "peeringLocation",
+            "in": "query",
+            "description": "The location of the peering.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "kind",
+            "in": "query",
+            "description": "The kind of the peering.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Direct",
+              "Exchange"
+            ],
+            "x-ms-enum": {
+              "name": "kind",
```

Could you please change the name of this enum type? We have named this differently in the previous api-version: https://github.com/Azure/azure-rest-api-specs/pull/17239/files
```suggestion
              "name": "LegacyPeeringsKind",
```

---

updated!